Assets/PatchKit/UnityEditorExtension/Editor/UI/SafeBuildAndUploadScreen.cs

```diff
@@ -280,6 +280,12 @@ private void DrawVersion()
                 "You haven't selected build location.",
                 MessageType.Error);
         }
+        else if (!IsScenesSelected)
+        {
+            EditorGUILayout.HelpBox(
+                "You haven't selected Scenes.",

```

Comment:
Powinno być `You haven't selected any scenes.`

Reply:
done